src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -654,7 +700,8 @@ subroutine SoilThermProp (bounds, num_urbanc, filter_urbanc, num_nolakec, filter
                   col%itype(c) /= icol_roof .and. col%itype(c) /= icol_road_imperv) .or. &
                   (col%itype(c) == icol_road_imperv .and. j > nlev_improad(l))) then
 
-                  satw = (h2osoi_liq(c,j)/denh2o + h2osoi_ice(c,j)/denice)/(dz(c,j)*watsat(c,j))
+                    ! TODO recalculate watsat and satw to have excess ice included
```

Comment:
We should make sure we track this TODO item.

Reply:
Made this an issue in #1819

Reply:
In the discussion in the issue we decided to remove the comment and leave the code as it is. I also searched for other places like this and didn't find any. And Matvey did previously as well, so we are good here.